### ISSUE TITLE
telemetry log on remote ssh generated under wrong path

### ISSUE THREAD

**Issue by edwardchenchen** [microsoft/vscode#163523]
Does this issue occur when all extensions are disabled?: 
with only remote ssh enabled

- VS Code Version: 
- OS Version: 
Version: 1.73.0-insider (user setup)
Commit: a8108049ab61b970f2ec1839dfb753054e07395e
Date: 2022-10-13T05:19:31.002Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19043
Sandboxed: Yes
Steps to Reproduce:

1. VS Code with Remote SSH, telemetry log is generated on the remote with path from local

Remote Linux
```
root@arm-11:~/c:/Users/Edward/AppData/Roaming/Code - Insiders/logs/20221013T010250/exthost1# pwd
/root/c:/Users/Edward/AppData/Roaming/Code - Insiders/logs/20221013T010250/exthost1
```

Windows
![image](https://user-images.githubusercontent.com/6397192/195533557-5afafb99-33d5-4647-8496-d1c5b4809758.png)

**Comment by lramos15** [comment]
@roblourens We utilize the `logsPath` argument to determine where to store logs. Is it possible that remote SSH is specifying an incorrect path when launching the server?

**Comment by roblourens** [comment]
I don't think this is Remote-SSH. We do set the `VSCODE_AGENT_FOLDER` env var to `$HOME/<a string from product.json>`, and it could be set wrong if the setting `remote.SSH.serverInstallPath` was set, did you set that setting @edwardchenchen? 

Sharing the log from the Remote-SSH output channel might help.

**Comment by roblourens** [comment]
And also, are there other log files/data under that C: path? I still doubt this is Remote-SSH though.

**Comment by edwardchenchen** [comment]
> 

only telemetry.log on linux

![image](https://user-images.githubusercontent.com/6397192/195966688-bd85972c-ad18-410b-aa65-2d1dda8d537a.png)

windows has more logs

![image](https://user-images.githubusercontent.com/6397192/195966703-7d828c81-bf10-46e9-8a29-c4f1a28e45b2.png)

**Comment by edwardchenchen** [comment]
> I don't think this is Remote-SSH. We do set the `VSCODE_AGENT_FOLDER` env var to `$HOME/<a string from product.json>`, and it could be set wrong if the setting `remote.SSH.serverInstallPath` was set, did you set that setting @edwardchenchen?
> 
> Sharing the log from the Remote-SSH output channel might help.

`remote.SSH.serverInstallPath` is not set

```
[14:21:45.551] Log Level: 2
[14:21:45.554] remote-ssh@0.90.1
[14:21:45.554] win32 x64
[14:21:45.576] SSH Resolver called for "ssh-remote+7b22686f73744e616d65223a224f5648227d", attempt 1
[14:21:45.577] "remote.SSH.useLocalServer": false
[14:21:45.577] "remote.SSH.showLoginTerminal": false
[14:21:45.577] "remote.SSH.remotePlatform": {
REMOVED}
[14:21:45.578] "remote.SSH.path": undefined
[14:21:45.578] "remote.SSH.configFile": undefined
[14:21:45.578] "remote.SSH.useFlock": true
[14:21:45.579] "remote.SSH.lockfilesInTmp": false
[14:21:45.579] "remote.SSH.localServerDownload": auto
[14:21:45.579] "remote.SSH.remoteServerListenOnSocket": false
[14:21:45.579] "remote.SSH.showLoginTerminal": false
[14:21:45.580] "remote.SSH.defaultExtensions": []
[14:21:45.580] "remote.SSH.loglevel": 2
[14:21:45.580] "remote.SSH.enableDynamicForwarding": true
[14:21:45.580] "remote.SSH.enableRemoteCommand": false
[14:21:45.580] "remote.SSH.serverPickPortsFromRange": {}
[14:21:45.580] "remote.SSH.serverInstallPath": {}
[14:21:45.586] SSH Resolver called for host: OVH
[14:21:45.586] Setting up SSH remote "OVH"
[14:21:45.595] Using commit id "7ef8e6b87a2a5a25ba1ef946bd1640ca3510956c" and quality "insider" for server
[14:21:45.599] Install and start server if needed
[14:21:45.623] Checking ssh with "ssh -V"
[14:21:45.670] > OpenSSH_for_Windows_8.1p1, LibreSSL 3.0.2

[14:21:45.677] Running script with connection command: ssh -T -D 56827 "OVH" bash
[14:21:45.679] Terminal shell path: C:\Windows\System32\cmd.exe
[14:21:46.437] > ]0;C:\Windows\System32\cmd.exe
[14:21:46.437] Got some output, clearing connection timeout
[14:21:46.838] > e004949014ae: running
[14:21:46.868] > Acquiring lock on /root/.vscode-server-insiders/bin/7ef8e6b87a2a5a25ba1ef946bd16
> 40ca3510956c/vscode-remote-lock.root.7ef8e6b87a2a5a25ba1ef946bd1640ca3510956c   
[14:21:46.887] > Found existing installation at /root/.vscode-server-insiders/bin/7ef8e6b87a2a5a2
> 5ba1ef946bd1640ca3510956c...
> Checking /root/.vscode-server-insiders/.7ef8e6b87a2a5a25ba1ef946bd1640ca3510956c
> .log and /root/.vscode-server-insiders/.7ef8e6b87a2a5a25ba1ef946bd1640ca3510956c
> .pid for a running server
> Looking for server with pid: 32270
[14:21:46.905] > Found running server...
> 
> *
> * Reminder: You may only use this software with Visual Studio family products, 
> * as described in the license (https://go.microsoft.com/fwlink/?linkid=2077057)
> *
> 
[14:21:46.922] > Checking server status on port 35117 with wget
> e004949014ae: start  
> SSH_AUTH_SOCK====    
> DISPLAY====
> webUiAccessToken==== 
> listeningOn==35117== 
> osReleaseId==ubuntu==
> arch==x86_64==       
> tmpDir==/run/user/0==
> platform==linux==    
> unpackResult====     
> didLocalDownload==0==
> downloadTime====     
> installTime====      
> extInstallTime====
> serverStartTime====
> connectionToken==a1a11111-a1a1-11aa-a11a-aa11a1a1aa11==
> e004949014ae: end
[14:21:46.922] Received install output:   
SSH_AUTH_SOCK====    
DISPLAY====
webUiAccessToken==== 
listeningOn==35117== 
osReleaseId==ubuntu==
arch==x86_64==       
tmpDir==/run/user/0==
platform==linux==    
unpackResult====     
didLocalDownload==0==
downloadTime====     
installTime====      
extInstallTime====
serverStartTime====
connectionToken==a1a11111-a1a1-11aa-a11a-aa11a1a1aa11==

[14:21:46.923] Remote server is listening on 35117
[14:21:46.923] Parsed server configuration: {"serverConfiguration":{"remoteListeningOn":{"port":35117},"osReleaseId":"ubuntu","arch":"x86_64","webUiAccessToken":"","sshAuthSock":"","display":"","tmpDir":"/run/user/0","platform":"linux","connectionToken":"a1a11111-a1a1-11aa-a11a-aa11a1a1aa11"},"installUnpackCode":""}
[14:21:46.930] Starting forwarding server. localPort 56834 -> socksPort 56827 -> remotePort 35117
[14:21:46.935] Forwarding server listening on 56834
[14:21:46.935] Waiting for ssh tunnel to be ready
[14:21:46.937] Tunneled 35117 to local port 56834
[14:21:46.937] Resolved "ssh-remote+7b22686f73744e616d65223a224f5648227d" to "127.0.0.1:56834"
[14:21:46.938] [Forwarding server 56834] Got connection 0
[14:21:46.940] > 
[14:21:46.948] ------




[14:21:46.961] [Forwarding server 56834] Got connection 1
[14:21:47.006] [Forwarding server 56834] Got connection 2
[14:21:48.319] [Forwarding server 56834] Got connection 3
[14:21:48.320] [Forwarding server 56834] Got connection 4
[14:21:49.352] [Forwarding server 56834] Got connection 5
[14:21:49.353] [Forwarding server 56834] Got connection 6
[14:21:49.354] [Forwarding server 56834] Got connection 7
[14:21:49.354] [Forwarding server 56834] Got connection 8
```
